### PR TITLE
SF-1153: Update tslint-config-groupby package version

### PR DESCRIPTION
**Pull request by JoeyDeol**
Bump tslint-config-groupby version to v1.1.0 and update tsconfig.json to accommodate tslint updates.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -15,7 +15,9 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "outDir": "dist",
-    "sourceMap": true
+    "sourceMap": true,
+    "allowUnusedLabels": false,
+    "noImplicitReturns": true
```
Comment (jmbtutor) [inline tsconfig.json]: Remove this. I don't think we had the equivalent of this in the tslint config.

File: tsconfig.json [hunk 2]
```diff
@@ -15,7 +15,9 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "outDir": "dist",
-    "sourceMap": true
+    "sourceMap": true,
+    "allowUnusedLabels": false,
```
Comment (jmbtutor) [inline tsconfig.json]: Isn't this already the default? https://www.typescriptlang.org/docs/handbook/compiler-options.html

File: tsconfig.json (hunk 1 above)
Comment (JoeyDeol) [reply tsconfig.json]: Will do.

File: tsconfig.json (hunk 2 above)
Comment (JoeyDeol) [reply tsconfig.json]: After review, we have decided to remove this all well.

**Pull request merged**